hedera-node/hedera-app/src/main/java/com/hedera/node/app/ServicesMain.java

```diff
@@ -90,10 +91,16 @@ public void run() {
     public static void main(final String... args) throws Exception {
         BootstrapUtils.setupConstructableRegistry();
         final var registry = ConstructableRegistry.getInstance();
-        new SwirldsPlatformBuilder()
-                .withName("Hedera")
-                .withMain(() -> new Hedera(registry))
-                .withNodeId(args != null && args.length > 0 ? Integer.parseInt(args[0]) : 0)
-                .buildAndStart();
+
+        final Hedera hedera = new Hedera(registry);
+        final NodeId selfId = args != null && args.length > 0 ? new NodeId(Integer.parseInt(args[0])) : new NodeId(0);
+
+        final PlatformBuilder builder = new PlatformBuilder(
+                Hedera.APP_NAME, Hedera.SWIRLD_NAME, hedera.getSoftwareVersion(), hedera::newState, selfId);
+
+        final SwirldsPlatform platform = builder.build();
```

Comment:
Why does it needs to be a SwirldsPlatform? I would assume Platform should be used here and services have no idea about the implementation

Reply:
The only reason it is `SwirldsPlatform` is because there is no `start()` method on the interface `Platform`. Of the following options, which of the following do you prefer?

- keep it the way it is
- `Platform extends Startable`
- `((Startable) platform).start()`
- Create a new interface `StartablePlatform extends Startable, Platform`

Reply:
`Platform extends Startable` <- Maybe that is the best idea for now, what do you think?

Reply:
I'm ok with this solution. Done.